anybox/recipe/odoo/utils.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 
 
-MAJOR_VERSION_RE = re.compile(r'(\d+)[.](saas~|)(\d*)(\w*)')
+MAJOR_VERSION_RE = re.compile(r'(saas~)?(\d+)[.](saas~|)(\d*)(\w*)')
```

Comment:
Maybe it is good to add a paragraph of documentation about the new SaaS versions [here](https://github.com/anybox/anybox.recipe.odoo/blob/master/anybox/recipe/odoo/utils.py#L74)